Tools/Scripts/run-mvt-tests

```diff
@@ -30,317 +30,355 @@ if os.path.isdir(os.path.join(scripts_dir, 'webkitpy')):
+            test_name = test_entry['name']
+            test_result = test_entry['status']
+            if self.should_update_expectations:
+                self._store_test_result_this_run(test_suite, test_name, test_result)
+            if self._is_unexpected_failure(test_suite, test_name, test_result):
+                unexpected_failed += 1
+                _log.warning(f'[UNEXPECTED_FAIL] Test "{test_name}" from "{test_suite}" has unexpected result: {test_result}')
+                _log.info(test_entry['log'])
+            elif test_result != 'passed':
+                _log.info(f'[FAIL][EXPECTED] {test_name}')
+                expected_failed += 1
+            else:
+                _log.info(f'[PASS] {test_name}')
+        return total_tests, unexpected_failed, expected_failed
+
+    def maybe_update_expectations(self):
+        if self.should_update_expectations and self._results_this_run:
+            with open(self._expectations_file_path, 'w') as fd:
+                fd.write(self._json_compact_printer(self._results_this_run))
+                fd.write('\n')
+            _log.info(f'Expectations file updated at "{self._expectations_file_path}"')
+
+
+def run_mvt_test_suite(args):
+    _log.info(f'Starting WebDriver MVT runner with browser "{args.browser_name}" and platform "{args.platform}" configuration "{args.configuration}"')
+    mvtwebdriver_runner = MVTWebDriverRunner(args.platform, args.configuration.lower(), args.extra_browser_args, args.mvt_instance_address, args.browser_name)
+    mvtresultsexpectations_parser = MVTResultsExpectationsParser(args.platform, args.configuration, args.update_expectations)
+
+    # Run the test suite(s)
+    test_results = {}
+    suites = TEST_SUITES if args.suite is None else [args.suite]
+    num_suites = len(suites)
+    for suite in suites:
+        json_results = mvtwebdriver_runner.run_suite(suite)
+        tests_run, unexpected_failures, expected_failures = mvtresultsexpectations_parser.analyze_results(json_results)
+        test_results[suite] = [tests_run, unexpected_failures, expected_failures]
+        exit_code = unexpected_failures
+        tests_pass = tests_run - unexpected_failures - expected_failures
+        suite_summary_header_str = f"[Suite {suite}] Ran {tests_run} tests of which:"
+        _log.info('-' * len(suite_summary_header_str))
+        _log.info(suite_summary_header_str)
+        _log.info(f"    - {tests_pass} tests passed.")
+        if expected_failures > 0:
+            _log.info(f"    - {expected_failures} tests were expected failures.")
+        if unexpected_failures > 0:
+            _log.info(f"    - {unexpected_failures} tests were unexpected failures.")
+        _log.info('-' * len(suite_summary_header_str) + '\n')
+
+    # Print global summary if more than one suite was run
+    if num_suites > 1:
+        _log.info('######################')
+        _log.info('# TOTAL TEST SUMMARY #')
+        _log.info('######################')
```

Comment:
“Total test summary” sounds odd in English. I think you want to just write “Summary” here 😉 

```suggestion
        _log.info('###########')
        _log.info('# SUMMARY #')
        _log.info('###########')
```